Apps/Src/RTOSPedals.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "RTOSPedals.h"
+
+
+/**
+* @brief   Read both pedal percentage values and set the global variables
+* @param   None
+* @return  None
+*/ 
+void Task_ReadPedals(void *p_arg){
+    OS_ERR err;
+
+    while(1){
+        pedals.ACC_PERCENT = Pedals_Read(ACCELERATOR);
+        pedals.BRAKE_PERCENT = Pedals_Read(BRAKE);
+        // The velocity and lights threads can run without blocking
+        // semaphores, the delay added an the end of this loop will allow
+        // them to run
+
+        // Post to semaphores (velocityChange, lightsChange) 
+        // OSSemPost (&VelocityChange_Sem4, OS_OPT_POST_ALL, &err);
+        // OSSemPost (&LightsChange_Sem4, OS_OPT_POST_ALL, &err);
```

Comment:
Here, I would keep track of the previous value that this task read and compare if the new value is different from the previous value for either brake or accel. If either is different, then post the semaphores.